packages/records/lib/types.ts

```diff
@@ -38,6 +38,7 @@ export type ReturnedRecord = {
 } & Record<string, RecordValue> & { id: string };
 
 export type LastAction = 'ADDED' | 'UPDATED' | 'DELETED' | 'added' | 'updated' | 'deleted';
+export type CombinedFilterAction = `${LastAction},${LastAction}`;
```

Comment:
Those are duplicated type, not sure why we are not using https://github.com/types pkg here

Reply:
ah maybe, I will make a following PR to sync